src/main/webapp/js/pages/geppetto/GEPPETTO.Main.js

```diff
@@ -107,9 +108,9 @@ define(function (require) {
              * Initialize web socket communication
              */
             init: function () {
-            	var host = GEPPETTO.MessageSocket.protocol + window.location.host + '/' + GEPPETTO_CONFIGURATION.contextPath + '/GeppettoServlet';
+                var host = path.join(GEPPETTO.MessageSocket.protocol + window.location.host + window.location.pathname.substring(0,window.location.pathname.lastIndexOf("/")), GEPPETTO_CONFIGURATION.contextPath, "GeppettoServlet")
```

Comment:
Testing this on VFB this generates an incorrect websocket url to be used.

before this change: ws://localhost:8081/org.geppetto.frontend//GeppettoServlet
after this change: ws:/localhost:8081/org.geppetto.frontend/org.geppetto.frontend/GeppettoServlet

not sure by what window.location.pathname.substring(0,window.location.pathname.lastIndexOf("/")) should return, but unless we change the contextPath to "/" in the GeppettoConfiguration this will not work for me.